src/components/FocusTrap/WIDE_LAYOUT_INACTIVE_SCREENS.ts

```diff
@@ -1,7 +1,10 @@
 import NAVIGATORS from '@src/NAVIGATORS';
 import SCREENS from '@src/SCREENS';
 
-// Screens that should not have active focus trap when rendered on wide screen in order to allow Tab navigation in LHP and RHP
+/**
+ * Screens displayed in the BottomTab and CentralPane displayed side by side that should not have active
+ * focus trap when rendered on a wide screen to allow navigation between them using the keyboard
+ */
 const WIDE_LAYOUT_INACTIVE_SCREENS: string[] = [
```

Comment:
Only screens displayed in the BottomTab or CentralPane should be added to this array, FocusTrap is not active on these screens to allow navigation between LHN and CentralPane on a wide layout.

Keyboard navigation:  
- before adding 
`SCREENS.WORKSPACE.ACCOUNTING.ROOT` to `WIDE_LAYOUT_INACTIVE_SCREENS`:

https://github.com/user-attachments/assets/284cf436-65f4-4eef-82ee-6b4a113c5358

- after adding 
`SCREENS.WORKSPACE.ACCOUNTING.ROOT` to `WIDE_LAYOUT_INACTIVE_SCREENS`:

https://github.com/user-attachments/assets/9911d4a5-64e8-4919-a194-7c743f4174da

Reply:
Thanks for the comment Wojtek! I adjusted the code accordingly (removed and added some screen from this array)